src/main/java/io/cryostat/platform/internal/PodmanPlatformStrategy.java

```diff
@@ -111,7 +111,7 @@ public boolean isAvailable() {
     private boolean testPodmanApi() {
         CompletableFuture<Boolean> result = new CompletableFuture<>();
         URI requestPath = URI.create("http://d/info");
-        Executors.newSingleThreadExecutor()
+        ForkJoinPool.commonPool()
```

Comment:
Just a question: Does this switch to use commonPool part of the fix needed? If not, what is the purpose here?

Reply:
It's not part of the fix, it just reduces the overhead of maintaining a separate pool with a single thread in it for doing this work. May as well use the pool that the JVM already maintains, since these tasks are simple enough that they shouldn't really interfere with anything else - not forking a bunch of other subtasks or running at high frequency etc.